src/dist/replication/lib/replica_learn.cpp

```diff
@@ -749,10 +750,11 @@ void replica::on_copy_remote_state_completed(
             else
             {
                 derror(
-                    "%s: on_copy_remote_state_completed[%016llx]: learner = %s, learn duration = %" PRIu64 " ms, apply learned state from private log failed, err = %s, duration = %" PRIu64 " ns",
-                    name(), req.signature, req.learner.to_string(),
-                    err.to_string(),
+                    "%s: on_copy_remote_state_completed[%" PRIx64 "]: learner = %s, "
```

Comment:
we print signature in "%016llx" in all places, so that we can trace all logs of the same learning by it.